doc/classes/Transform2D.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<class name="Transform2D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+<class name="Transform2D" keywords="matrix" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
```

Comment:
It's true, but it's a bit of a stretch, isn't it? I don't see why anyone would look for "matrices" in Godot. **Basis** and **Projection** are matrices, too!

Reply:
~~They wouldn't even find it by searching "matrices" :sob:~~

Yeah the keyword would only matter for F1 search - online docs would likely find it via description text anyway.

Also you're right, I should've added it to those classes too, but forgot they're a thing while making this change...
As stated in my larger comment, I'm a fan of discoverability, hence I'd be in favor of adding the keywords to those classes I missed.
But at the same time, I'm trying to better understand the conservative perspective you're taking, of trying to not include keywords in certain places, so maybe you can convince me of the opposite?